Patch 2

Aaron-Fine opened the pull request:
Updated setup.md file to open port 8080, correct some typos, and add a final restart of Tomcat. 

NOTE: The Apache port forwarding doesn't work. going to http://ip.address redirects to http://ip.address/openstorefront/ but then says the site is unavailable. Going to http://ip.address:8080 shows the tomcat landing page. To get to the site you have to go to http://ip.address:8080/openstorefront 

Tested on CentOS 7 Minimal Install on a local VM.

dshurt commented:
Opening port 8080 is not likely what they should do.

What need to do is fix the apache setup.   (perhaps the mod-proxy module is not setup?)   The link with talks about checking that.   

Let check that first and see if that can be resolved then clarify instructions.

Aaron-Fine commented:
I fixed the issue on my local machine and updated the file accordingly. The problem was with SELinux blocking outbound connections. 

It was a simple one line fix, but finding that one line was the challenge! I also tweaked the VirtualHost code to more closely match the samples provided with the apache documentation (adding quotation marks and trailing /'s)
